src/ui/render.rs

```diff
@@ -295,6 +295,7 @@ fn draw_spreadsheet(f: &mut Frame, app_state: &AppState, area: Rect) {
     let table = Table::new(
         // Combine header and data rows
         std::iter::once(header).chain(rows),
+        [Constraint::Length(1)]
```

Comment:
`Table::new` now requires two args. Hard-coding a single `Constraint::Length(1)` is me being lazy. I have no idea whether this is OK.